resources/views/platform/components/game/current-progress/stats.blade.php

```diff
@@ -0,0 +1,84 @@
+@props([
+    'totalAchievementsCount' => 0,
+    'totalPointsCount' => 0,
+    'numEarnedSoftcoreAchievements' => 0,
+    'numEarnedHardcoreAchievements' => 0,
+    'numEarnedSoftcorePoints' => 0,
+    'numEarnedHardcorePoints' => 0,
+    'numEarnedWeightedPoints' => 0,
+])
+
+<?php
+$hasHardcoreProgress = $numEarnedHardcoreAchievements > 0;
+$hasSoftcoreProgress = $numEarnedSoftcoreAchievements > 0;
+?>
+
+<div>
+    @if ($hasHardcoreProgress && !$hasSoftcoreProgress)
+        <p>
+            <span class="font-bold">{{ localized_number($numEarnedHardcoreAchievements) }}</span>
+            of {{ localized_number($totalAchievementsCount) }} achievements
+        </p>
+
+        @if ($totalPointsCount > 0)
+            <div class="flex gap-x-1">
+                <div class="flex font-bold gap-x-1">
+                    {{ localized_number($numEarnedHardcorePoints) }}
+                    <x-points-weighted-container>
+                        ({{ localized_number($numEarnedWeightedPoints) }})
+                    </x-points-weighted-container>
+                </div>
+
+                <p>
+                    of {{ localized_number($totalPointsCount) }} points
+                </p>
+            </div>
+        @endif
+    @elseif ($hasSoftcoreProgress && !$hasHardcoreProgress)
+        <p>
+            <span class="font-bold">{{ localized_number($numEarnedSoftcoreAchievements) }}</span>
+            of {{ localized_number($totalAchievementsCount) }} softcore achievements
+        </p>
+
+        @if ($totalPointsCount > 0)
+            <div class="flex gap-x-1">
+                <div class="flex font-bold gap-x-1">
+                    {{ localized_number($numEarnedSoftcorePoints) }}
+                </div>
+
+                <p>
+                    of {{ localized_number($totalPointsCount) }} softcore points
+                </p>
+            </div>
+        @endif
+    @elseif ($hasHardcoreProgress && $hasSoftcoreProgress)
+        <p>
+            <span class="font-bold">{{ localized_number($numEarnedHardcoreAchievements) }}</span>
+            hardcore {{ mb_strtolower(__res('achievement', $numEarnedHardcoreAchievements)) }}
```

Comment:
Minor: when user has hardcore and softcore, the "of X" isn't displayed in the achievement list. I agree it shouldn't be on both lines, but I think it would make sense to still show it on the hardcore line.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/46c1e056-97b2-4b9d-9909-2cfb8b664e99)

vs
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/b5cbf17d-bfb1-4564-93c8-2833bd359650)

Reply:
In latest, I've added the "of N" for the hardcore achievement count, as shown in your 2nd screenshot.
Also matching your 2nd screenshot, we don't have an "of N" for the hardcore points. I attempted to add that too, but there isn't enough space once the RetroPoints value reaches a very high number.

![Screenshot 2023-08-26 at 1 00 07 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/83da76fd-f374-47f8-91b6-6be8c38d26ba)

Reply:
I didn't even notice the "of N" hardcore points was missing. I guess it's not that important to me.

Reply:
It's probably the blaring white retro points disrupting the chain of thought, so my brain ignores the "of N" on a fully hardcore/softcore game.